### PR TITLE
Update hyperscale-architecture.md

### PR DESCRIPTION
**Pull request by oslake**

### CODE REVIEW

**Comment by oslake** [comment]
#sign-off

File: azure-sql/database/hyperscale-architecture.md [hunk 1]
```diff
@@ -34,12 +34,14 @@ A Hyperscale database contains the following types of components: compute nodes,
 
 ## Compute
 
-The compute node is where the relational engine lives. The compute node is where language, query, and transaction processing occur. All user interactions with a Hyperscale database happen through compute nodes.
+The compute node is where the relational engine lives. The compute node is where language, query, and transaction processing occur. All user interactions with a Hyperscale database happen through compute nodes.  Computes nodes can either be configured to use serverless or provisioned compute.
```
Comment (dimitri-furman) [inline azure-sql/database/hyperscale-architecture.md]:
```suggestion
The compute node is where the relational engine lives. The compute node is where language, query, and transaction processing occur. All user interactions with a Hyperscale database happen through compute nodes.  Compute nodes can either be configured to use serverless or provisioned compute.
```
small typo

**Comment by Court72** [comment]
@dimitri-furman 

Thanks for reviewing the proposed changes. 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by dimitri-furman — APPROVED** [review]

**Comment by dimitri-furman** [comment]
#sign-off

**Pull request merged**